python/feathub/feature_views/sliding_feature_view.py

```diff
@@ -270,46 +277,88 @@ def _validate(features: Sequence[Union[str, Feature]]) -> None:
 
         # Per-row transform features listed before the first SlidingWindowTransform
         # feature.
-        per_row_transform_feature_names: Set[str] = set()
-        sliding_window_transforms_group_by_keys: Set[str] = set()
+        pre_sliding_per_row_feature_names: Set[str] = set()
+
+        # Per-row transform features listed after the first SlidingWindowTransform
+        # feature.
+        post_sliding_per_row_features: Set[Feature] = set()
```

Comment:
Would it be more readable to use either `Set[str]` or `Set[Feature]` for both sets above?